src/javascript/binary/pages/resources/trading_timesws.js

```diff
@@ -0,0 +1,156 @@
+var TradingTimesWS = (function() {
+    "use strict";
+
+    var $date      = $('#trading-date');
+    var $container = $('#trading-times');
+    var columns    = ['Asset', 'Opens', 'Closes', 'Settles', 'UpcomingEvents'];
+
+
+    var init = function() {
+        showLoadingImage($container);
+        sendRequest('today');
+
+        $date.val(moment.utc(new Date()).format('YYYY-MM-DD'));
+        $date.datepicker({minDate: 0, maxDate: '+1y', dateFormat: 'yy-mm-dd', autoSize: true});
+        $date.change(function() {
+            $container.empty();
+            showLoadingImage($container);
+            sendRequest();
+        });
+    };
+
+    var sendRequest = function(date) {
+        BinarySocket.send({"trading_times": (date ? date : $date.val())});
+    };
+
+    var populateTable = function(response) {
+        $('#errorMsg').addClass('hidden');
+
+        var markets = response.trading_times.markets;
+
+        var $ul = $('<ul/>');
+        var $contents = $('<div/>');
+
+        for(var m = 0; m < markets.length; m++) {
+            var tabID = 'tradingtimes-' + markets[m].name.toLowerCase();
+
+            // tabs
+            $ul.append($('<li/>', {class: 'ja-hide'}).append($('<a/>', {href: '#' + tabID, text: markets[m].name, id: 'outline'})));
```

Comment:
For JA language we hide all markets except Forex major pair using ja-hide.
Another way is to remove all other markets with JS instead of hiding them with css.